python/test/unit/operators/test_matmul.py

```diff
@@ -108,7 +117,7 @@ def kernel(Y, X, N, BLOCK_SIZE: tl.constexpr):
         ],
     ),
 )
-def test_op(BLOCK_M, BLOCK_N, BLOCK_K, SPLIT_K, NWARP, NSTAGE, M, N, K, AT, BT, ADTYPE, BDTYPE, ALLOW_TF32, F8_FASTACCUM):
+def test_op(BLOCK_M, BLOCK_N, BLOCK_K, SPLIT_K, NWARP, NSTAGE, M, N, K, AT, BT, ADTYPE, BDTYPE, ALLOW_TF32, F8_FASTACCUM, DOT_OUT_DTYPE, C_DTYPE):
```

Comment:
I would rather replace `DOT_OUT_DTYPE` with `ACCUMULATION_DTYPE` or something like that. Otherwise it's confusing, since `dot` often refers to the entire matmul as well.

Reply:
Also, `C_DTYPE` is a bit confusing as well, since it sort of assumes that we'll call the output of our `tt.ops.dot()` call "C". Maybe just call it `OUTPUT_DTYPE`.